sympy/polys/polytools.py

```diff
@@ -4551,6 +4570,32 @@ def degree_list(f, *gens, **args):
 
     return tuple(map(Integer, degrees))
 
+@public
```

Comment:
Just asking for info, what's this `public` for?

Reply:
This public is to make this function public i.e we can call it directly just like other public functions like `degree_list` and other polytools.

Reply:
thanks for the explanation.  please resolve this conversation.